daemon/cmd/daemon_main.go

```diff
@@ -1087,6 +1087,9 @@ func initializeFlags() {
 	flags.Duration(option.IPAMCiliumNodeUpdateRate, 15*time.Second, "Maximum rate at which the CiliumNode custom resource is updated")
 	option.BindEnv(Vp, option.IPAMCiliumNodeUpdateRate)
 
+	flags.Bool(option.EnableK8sNetworkPolicy, defaults.EnableK8sNetworkPolicy, "Enable the K8s NetworkPolicy")
```

Comment:
Consistency with the other descriptions, plus "the" is probably not needed.
```suggestion
	flags.Bool(option.EnableK8sNetworkPolicy, defaults.EnableK8sNetworkPolicy, "Enable support for K8s NetworkPolicy")
```